lib/netconf/odyssey/genesis.json

```diff
@@ -0,0 +1,636 @@
+        "default_send_enabled": false
+      },
+      "balances": [
+        {
+          "address": "story1kz2cgurfhgege6zqnk8vwupwzqcjry0dj99f7f",
+          "coins": [
+            {
+              "denom": "stake",
+              "amount": "10000000000000000"
+            }
+          ]
+        },
+        {
+          "address": "story1wl873hxdkhhjalt86zgmyphfzlt2kl39cxp8pk",
+          "coins": [
+            {
+              "denom": "stake",
+              "amount": "10000000000000000"
+            }
+          ]
+        },
+        {
+          "address": "story1pdj0z84lau0l7vf2jl4qs7yggv48p3av2g732g",
+          "coins": [
+            {
+              "denom": "stake",
+              "amount": "10000000000000000"
+            }
+          ]
+        },
+        {
+          "address": "story1z0uqv8xrw2j23mlcpmzatt3jj9hzezdqpgfzvq",
+          "coins": [
+            {
+              "denom": "stake",
+              "amount": "10000000000000000"
+            }
+          ]
+        },
+        {
+          "address": "story10j30q04u6hs0ktcxlwh77j22cp7v26gktq8ry7",
+          "coins": [
+            {
+              "denom": "stake",
+              "amount": "10000000000000000"
+            }
+          ]
+        },
+        {
+          "address": "story19stnkys77a97pxg9uqd2j24sljun4nzhwa6wxc",
+          "coins": [
+            {
+              "denom": "stake",
+              "amount": "10000000000000000"
+            }
+          ]
+        },
+        {
+          "address": "story12zzvtjr4yyf0qfsyf5eayrrftjx6vzukqnuxpm",
+          "coins": [
+            {
+              "denom": "stake",
+              "amount": "10000000000000000"
+            }
+          ]
+        },
+        {
+          "address": "story1a6u7vn2j5e6z0l6ykdyv809gxq0qaywqgu9zxy",
+          "coins": [
+            {
+              "denom": "stake",
+              "amount": "10000000000000000"
+            }
+          ]
+        }
+      ],
+      "supply": [
+        {
+          "denom": "stake",
+          "amount": "80000000000000000"
+        }
+      ],
+      "denom_metadata": [],
+      "send_enabled": [{ "denom": "stake", "enabled": true }]
+    },
+    "distribution": {
+      "params": {
+        "community_tax": "0.000000000000000000",
+        "base_proposer_reward": "0.000000000000000000",
+        "bonus_proposer_reward": "0.000000000000000000",
+        "withdraw_addr_enabled": true
+      },
+      "fee_pool": {
+        "community_pool": []
+      },
+      "delegator_withdraw_infos": [],
+      "previous_proposer": "",
+      "outstanding_rewards": [],
+      "validator_accumulated_commissions": [],
+      "validator_historical_rewards": [],
+      "validator_current_rewards": [],
+      "delegator_starting_infos": [],
+      "validator_slash_events": []
+    },
+    "evmengine": {
+      "params": {
+        "execution_block_hash": "9evG8Jgui/f9Uys/lZ+E0Ss97dKCevjTH1OJRHvtr8Y="
```

Comment:
It would be helpful to add linked PR in story-geth.
maybe https://github.com/piplabs/story-geth/pull/29?

Reply:
++